docs/_layouts/base.html

```diff
@@ -2,4 +2,4 @@
 <link rel="preconnect" href="https://cdnjs.cloudflare.com" crossorigin>
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 
-{{ content }}
```

Comment:
Is this intentional?

Reply:
Not at all, I deleted the last line due to changes, corrected 👍